### PR TITLE
[v1.x] ONNX Supoort for MXNet reverse op

### PR DESCRIPTION
**Pull request by Zha0q1**
There is no direct mapping of reverse in the onnx op set so I had to mimic the behavior by 
1) transpose the axis to be reversed to the first dim
2) Gather along fist dim in reversed order
3) transpose first axis back to its old position

The performance is not great most likely, but functionally i got it to behave the same as mxnet reverse

### CODE REVIEW

File: python/mxnet/contrib/onnx/mx2onnx/_op_translations.py [hunk 1]
```diff
@@ -2784,6 +2784,48 @@ def convert_arange(node, **kwargs):
...
+    """
+    from onnx.helper import make_node
+    name, input_nodes, attrs = get_inputs(node, kwargs)
+
+    axis = int(attrs.get('axis', 0))
+
+    # Transpose takes perm as a parameter, so we must 'pad' the input to a known dim (10 here)
+    perm = [i for i in range(10)]
+    perm[0], perm[axis] = axis, 0
+    print(perm)
```
Comment (josephevans) [inline python/mxnet/contrib/onnx/mx2onnx/_op_translations.py:2799]: remove debug?

File: python/mxnet/contrib/onnx/mx2onnx/_op_translations.py [hunk 2]
```diff
@@ -2784,6 +2784,48 @@ def convert_arange(node, **kwargs):
...
+def convert_reverse(node, **kwargs):
+    """Map MXNet's reverse operator attributes to ONNX
+    """
+    from onnx.helper import make_node
+    name, input_nodes, attrs = get_inputs(node, kwargs)
+
+    axis = int(attrs.get('axis', 0))
+
+    # Transpose takes perm as a parameter, so we must 'pad' the input to a known dim (10 here)
+    perm = [i for i in range(10)]
```
Comment (josephevans) [inline python/mxnet/contrib/onnx/mx2onnx/_op_translations.py:2797]: what happens if 10 is not enough?

Comment (Zha0q1) [reply python/mxnet/contrib/onnx/mx2onnx/_op_translations.py:2797]: I think in mxnet 10-d is the largest you can get

Comment (Zha0q1) [reply python/mxnet/contrib/onnx/mx2onnx/_op_translations.py:2797]: I just checked: it seems we can create > 10-d tensors, but I think many ops do not support >= 10d tensors and in general there is no use case for high-dimensional tensors

Comment (josephevans) [reply python/mxnet/contrib/onnx/mx2onnx/_op_translations.py:2797]: Got it, thanks for the explanation.

**Review by josephevans — APPROVED** [review]
LGTM, thanks!

**Review by waytrue17 — APPROVED** [review]
LGTM, thanks!

**Pull request merged**